### PR TITLE
wasmer: init at 0.13.0

### PR DESCRIPTION
**Pull request by Br1ght0ne**
###### Motivation for this change
https://github.com/wasmerio/wasmer/releases/tag/0.13.0

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---
This change is [Reviewable](https://reviewable.io/reviews/nixos/nixpkgs/77363)

### CODE REVIEW

File: pkgs/development/interpreters/wasmer/default.nix [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    owner = "wasmerio";
+    repo = pname;
+    rev = version;
+    sha256 = "045pldbgfbcv8071qs856sh9fpyzpyr5ybnsxswkyy2xsczi2xa8";
+    fetchSubmodules = true;
+  };
+
+  cargoSha256 = "1gydvj9f62qf1cfwpjrjcg3cwvh0byj91wnmid2flj2if5mnij9d";
+
+  nativeBuildInputs = [ cmake llvmPackages.libclang openssl pkgconfig ];
```
Comment (jonringer) [inline pkgs/development/interpreters/wasmer/default.nix]: why is openssl needed? for the command line tool?

**Review by jonringer — APPROVED** [review]
LGTM

```
[4 built, 13 copied (417.1 MiB), 91.7 MiB DL]
https://github.com/NixOS/nixpkgs/pull/77363
1 package were built:
wasmer
```

**Comment by jonringer** [comment]
@GrahamcOfBorg build wasmer

File: pkgs/development/interpreters/wasmer/default.nix [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{ lib, rustPlatform, fetchFromGitHub, cmake, llvmPackages, openssl, pkgconfig }:
```
Comment (dywedir) [inline pkgs/development/interpreters/wasmer/default.nix]: Nitpick: please use `pkg-config` (see https://github.com/NixOS/nixpkgs/pull/55094/commits/bff54052f040d57d8a3f4b43e8960497c311f04b)

File: pkgs/development/interpreters/wasmer/default.nix (hunk 1 above)
Comment (filalex77) [reply pkgs/development/interpreters/wasmer/default.nix]: I think it's not needed, after some testing. Removed.

File: pkgs/development/interpreters/wasmer/default.nix (hunk 2 above)
Comment (filalex77) [reply pkgs/development/interpreters/wasmer/default.nix]: Done.

**Comment by jonringer** [comment]
@GrahamcOfBorg build wasmer

**Pull request merged**